websocket/src/main/java/com/sun/ts/tests/websocket/api/jakarta/websocket/clientendpointconfig/WSClientIT.java

```diff
@@ -0,0 +1,1094 @@
+					int size_actual = tmpp.size();
+					if (size_actual == expected_subprotocols.size()) {
+						for (String subpro : tmpp) {
+							if (expected_subprotocols.contains(subpro)) {
+								log.append("sub protocol " + subpro + " found is expected|");
+							} else {
+								passed = false;
+								log.append("sub protocol " + subpro + " found is not expected|");
+							}
+						}
+					} else {
+						passed = false;
+						log.append("getPreferredSubprotocols() returned not exactly the same size.");
+						log.append("Expecting seven, returned " + size_actual);
+					}
+				} else {
+					log.append("getPreferredSubprotocols() return empty List|");
+					passed = false;
+				}
+			} else {
+				passed = false;
+				log.append("getPreferredSubprotocols() return null list");
+			}
+
+			if (tmpd != null) {
+				log.append("getDecoders() return non-null List|");
+				if (!tmpd.isEmpty()) {
+					int size = tmpd.size();
+					log.append("getDecoders() return non-empty List  with size " + tmpd.size() + "|");
+					if (size == 2) {
+						for (Class<? extends Decoder> decoder : tmpd) {
+							log.append("Decoder " + decoder + " found ");
+							if (expected_decoders.contains(decoder)) {
+								log.append("is expected.|");
+							} else {
+								passed = false;
+								log.append("is not expected.|");
+							}
+						}
+					} else {
+						passed = false;
+						log.append("getDecoders() returned not exactly the same size.");
+						log.append("Expecting two, returned " + size);
+					}
+				} else {
+					log.append("getDecoders() return empty List|");
+					passed = false;
+				}
+			} else {
+				passed = false;
+				log.append("getDecoders() return null list|");
+			}
+
+			List<Class<? extends Encoder>> tmpen = dcc.getEncoders();
+			if (tmpen != null) {
+				log.append("getEncoders() return non-null List");
+				if (!tmpen.isEmpty()) {
+					int size = tmpen.size();
+					log.append("getEncoders() return non-empty List  with size " + size + "|");
+					if (size == 2) {
+						for (Class<? extends Encoder> encoder : tmpen) {
+							log.append("Encoder " + encoder + " found ");
+							if (expected_encoders.contains(encoder)) {
+								log.append("is expected.|");
+							} else {
+								passed = false;
+								log.append("is not expected.|");
+							}
+						}
+					} else {
+						passed = false;
+						log.append("getEncoders() returned not exactly the same size.");
+						log.append("Expecting two, returned " + size);
+					}
+				} else {
+					log.append("getEncoders() return empty List|");
+					passed = false;
+				}
+			} else {
+				passed = false;
+				log.append("getEncoders() return null list");
+			}
+		} catch (Exception e) {
+			logger.log(Logger.Level.ERROR,e.getMessage() + log.toString());
+			throw new Exception(e);
+		}
+
+		if (passed == false) {
+			throw new Exception("Test failed: " + log.toString());
+		}
+	}
+
+	public void cleanup() {
+	}
+}
+
+class TCKConfigurator extends Configurator {
```

Comment:
this  looks un common syntax for definition of a class. Should it be in a separate file or just a static internal class.
Remove `}` from line 1083 and move it at the end

Reply:
code is same as https://github.com/jakartaee/platform-tck/blob/master/src/com/sun/ts/tests/websocket/api/jakarta/websocket/clientendpointconfig/WSClient.java#L1158